src/docfx/build/Build.cs

```diff
@@ -36,7 +36,7 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Erro
             string locale,
             CommandLineOptions options,
             ErrorLog errorLog,
-            RestoreMap restoreMap,
+            RestoreGitMap restoreMap,
```

Comment:
change the parameter to `restoreGitMap` as well?

Reply:
sounds good.